ports/lib3mf/vcpkg.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "lib3mf",
+  "version": "2.3.2",
+  "description": "lib3mf is an implementation of the 3D Manufacturing Format file standard",
+  "homepage": "https://github.com/3MFConsortium/lib3mf",
+  "license": "BSD-2-Clause",
+  "supports": "(windows & (x86 | x64) & !static & !staticcrt) | (linux & x64) | (osx & (x64 | arm64))",
+  "dependencies": [
+    "cpp-base64",
+    {
+      "name": "fast-float",
+      "version>=": "6.0.0"
+    },
+    {
+      "name": "libzip",
+      "version>=": "1.10.1"
+    },
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    "vcpkg-cmake-config",
+    {
+      "name": "zlib",
+      "version>=": "1.3.1"
+    }
```

Port manifest normally don't use version constraints. `vcpkg-cmake-config` is a host port.
```suggestion
    "fast-float",
    "libzip",
    {
      "name": "vcpkg-cmake",
      "host": true
    },
    {
      "name": "vcpkg-cmake-config",
      "host": true
    },
    "name": "zlib"
```

---

This defines minimum versions, not pinned versions, for users of manifest mode.
And it constrains how informed users mix ports. They might (and should) do their own testing. It is valuable information which version you tested/recommend, but it doesn't imply earlier (patch) versions wouldn't work.

vcpkg CI will only test the combination of latest versions.